### PR TITLE
Adding APlus?

### PR DESCRIPTION
**Pull request by arnowrld**
This is the name of Sunoco's convenience store chain & franchise.

### CODE REVIEW

**Comment by UKChris-osm** [comment]
Hi @arnowrld - it looks like APlus has it's own [Wikidata](https://www.wikidata.org/wiki/Q4646934) / [Wikipedia](https://en.wikipedia.org/wiki/A-Plus), so I think using these would be more suitable for this PR.

File: data/brands/shop/convenience.json [hunk 1]
```diff
@@ -335,6 +335,17 @@
         "shop": "convenience"
       }
     },
+    {
+      "displayName": "APlus (Sunoco)",
+      "id": "aplus-02op54",
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "brand": "APlus (Sunoco)",
+        "brand:wikidata": "Q1423218",
```
Comment (LaoshuBaby) [inline data/brands/shop/convenience.json:344]:
```suggestion
        "brand:wikidata": "Q4646934",
        "brand:wikipedia":"en:A-Plus",
```

File: data/brands/shop/convenience.json [hunk 2]
```diff
@@ -335,6 +335,17 @@
...
+        "name": "APlus (Sunoco",
```
Comment (LaoshuBaby) [inline data/brands/shop/convenience.json:345]:
```suggestion
        "name": "APlus (Sunoco)",
```

**Comment by bhousel** [comment]
Can we just use `name=APlus` and `brand=APlus` for these?
https://www.sunocolp.com/retail-solutions/aplus-convenience-stores

**Pull request merged**